tracer/test/test-applications/integrations/Samples.WebRequest/RequestHelpers.cs

```diff
@@ -551,7 +551,10 @@ private static void BeginGetResponse(bool tracingDisabled, string testName, stri
             HttpWebRequest request = (HttpWebRequest)System.Net.WebRequest.Create(GetUrlForTest(testName, url));
             request.Method = "POST";
             request.ContentLength = 1;
-            request.AllowWriteStreamBuffering = false;
+
+            // Prior to .NET 9, this toggle does nothing, but it defaults to "true"
+            // in .NET 9, setting this to false changes the behaviour significantly
+            request.AllowWriteStreamBuffering = true;
```

Comment:
😬

Reply:
Yeah... this isn't great 😅 I'm going to work on this one next ASAP, testing both `true` _and_ `false`, and adding support for the new implementation in .NET 9